explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -66,7 +68,9 @@ object SpectroscopyModesTable {
   type ColId = NonEmptyString
 
   implicit val reuseProps: Reusability[Props] =
-    Reusability.by(x => (x.scienceConfiguration, x.spectroscopyRequirements, x.constraints))
+    Reusability.by(x =>
+      (x.scienceConfiguration, x.spectroscopyRequirements, x.constraints, x.targets)
+    )
```

Comment:
Can we turn this into a `Reusability.derive` and use `Reusability.always` for `SpectroscopyModesMatrix`? Or receive a `Reusable[SpectroscopyModesMatrix]`.